lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -373,7 +373,13 @@ definitions:
     required:
       - next_page_token
       - subscribers
+      - total_count
     properties:
+      total_count:
+        type: integer
+        format: int64
+        description: total number of subscriber entries
```

Comment:
Fine with leaving it as `total_count`, but then let's note the fact that it's an estimation, or best-effort count, in the description